### DISCUSSION TITLE
OPT-060-Lot and BT-070-Lot

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#603] [Q&A] [answered]
Hi Lab,
still on sdk 1.7  
in notice 7 but I think in others as well , there is a strange situation in REQUIREMENTS FOR CONTRACT EXECUTION

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/993076c1-622d-44cf-96f4-23160cbbdd79)

OPT-060-Lot has got :  "presetValue" : "performance", and BT-070-Lot is mandatory , but with both I get a validation error : 
id : BR-BT-00070-0012
location : /pin:PriorInformationNotice/cac:ProcurementProjectLot/cac:TenderingTerms/cac:ContractExecutionRequirement[4]
failedAssertText : BT-70-Lot is mandatory in notice type '7' except under following condition: OPT-060-Lot is not present
diagnostic : BT-70-Lot
see : field:BT-70-Lot

Of course If I remove the presetValue from OPT-060-Lot,  notice is CVS valid
Is there a discrepancy between the rule and the OPT-060-Lot  ( or BT-070-Lot ) definition ?
Thanks and Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
When  OPT-060-Lot is set to "performance", then BT-70-Lot is mandatory, otherwise it is not allowed.
When removing OPT-060-Lot (which allows the identification of BT-70-Lot), you should also remove the content of BT-070-Lot.
For the rule to fire, both elements have to be under the same parent as shown here: https://github.com/OP-TED/eForms-SDK/discussions/568#discussioncomment-6438908
In absence of OPT-060-Lot, the rule may not get triggered.
